internal/cmd/destinations.go

```diff
@@ -87,11 +87,11 @@ func newDestinationsRemoveCmd() *cobra.Command {
 				return err
 			}
 
-			if len(destinations) == 0 {
+			if len(destinations.Items) == 0 {
```

Comment:
This is a common pattern. The resposne object should probably have a `Count` field